src/main/java/core/basesyntax/WorkWithFile.java

```
@@ -1,7 +1,67 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY_OPERATION = "supply";
+    private static final String BUY_OPERATION = "buy";
+    private static final String RESULT_OPERATION = "result";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        writeFile(toFileName, readFile(fromFileName));
+    }
+
+    private int[] readFile(String fileName) {
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fileName))) {
+            String inputLine = bufferedReader.readLine();
+
+            int bought = 0;
+            int supplied = 0;
+
+            while (inputLine != null) {
+                String[] splitLine = inputLine.split(",");
+
+                if (splitLine[0].equals(SUPPLY_OPERATION)) {
+                    supplied = Integer.parseInt(splitLine[1]) + supplied;
+                } else {
+                    bought = Integer.parseInt(splitLine[1]) + bought;
+                }
+
+                inputLine = bufferedReader.readLine();
+            }
+
+            int resultNumber = supplied - bought;
+            return new int[]{supplied, bought, resultNumber};
+        } catch (FileNotFoundException e) {
+            throw new RuntimeException("Can't find file", e);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file", e);
+        }
```

This violates checklist item #11: 'Don't ignore exceptions'. The checklist suggests re-throwing a `RuntimeException` with a more informative message that includes the file name, for example: `throw new RuntimeException("Can't read data from file " + fileName, e);`. This makes debugging much easier.